deploy/hpp.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: hpp
```

Comment:
I was using "hpp" as a quick and easy abbreviation, we may want to use a more official name here; like "kubevirt-hostpath" or "kubevirt-hostpath-provisioner" etc... what do you think?

Reply:
Done changed.